maestro-client/src/main/java/maestro/drivers/WebDriver.kt

```diff
@@ -168,6 +171,8 @@ class WebDriver(val isStudio: Boolean) : Driver {
     override fun contentDescriptor(excludeKeyboardElements: Boolean): TreeNode {
         ensureOpen()
 
+        detectWindowChange()
```

Comment:
Whenever focus shifts to a new tab, how can we shift focus back?

Reply:
Unfortunately there is no way to detect that at the moment, to the best of my knowledge. Selenium does not report what the currently active tab is.

What we could do (later) is to allow user to switch to a preferred tab manually.

Reply:
Sounds reasonable, thanks Dima!